package.json

```diff
@@ -22,8 +22,9 @@
     "test:watch": "jest --collectCoverage=false --watch",
     "test:unit": "jest --collectCoverage=false",
     "test:e2e": "playwright test",
+    "test:e2e-ci": "start-server-and-test run-server http://localhost:8443 test:e2e",
     "run-server": "cross-env nuxt build && pnpm start",
-    "e2e:ci": "start-server-and-test run-server http://localhost:8443 test:e2e",
+    "e2e:ci": "cross-env API_URL=http://localhost:3000 node proxy.js & TALKBACK_PID=$! && sleep 1 && pnpm run test:e2e-ci ; kill -15 $TALKBACK_PID",
```

Comment:
We might as well not use `cross-env` here, or really just use a proper bash script. This shouldn't get run outside of the CI so it doesn't need to be cross-platform compatible and the `&`, `&&` and `kill` commands I don't believe are cross-platform anyway.